tests/api_test.py

```diff
@@ -2966,8 +2966,14 @@ def test_vmap_caching(self):
 
 class RematTest(jtu.JaxTestCase):
 
-  def test_remat_basic(self):
-    @api.remat
+  @parameterized.named_parameters(
+      {"testcase_name": f"{suffix}", "remat": remat}
+      for suffix, remat in [
```

Comment:
Is it possible to factor this out?

Reply:
As in, stop repeating this same decorator text verbatim? Yeah, good idea, let me try...

Reply:
Done!

Reply:
Aw man, looks like there's a bad interaction with pytest-xdist or absl:

![image](https://user-images.githubusercontent.com/1458824/130160660-fd3fd8de-81f2-4835-b681-8d403bfef371.png)

Not sure what's up with that, but I don't really feel like debugging it...